### PR TITLE
iOS Adyen 2.8.4 + Xcode 10.2

### PR DESCRIPTION
**Pull request by mkharibalaji**
Updated Adyen iOS 2.8.4 + XCode 10.2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +55,65 @@ To give you as much flexibility as possible, Adyen SDK can be integrated via thi
...
+     return adyenHandled
+  }
+}
+```
+* AppDelegate.m file add the below function
+```
+.....
+- (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDictionary<NSString *,id> *)options {
+  
+  BOOL handledAdyen =[AdyenObjectiveCBridge applicationDidOpenURL:url];
```
Comment (bojanjukic) [inline README.md:112]: Use of undeclared identifier 'AdyenObjectiveCBridge'

Comment (mkharibalaji) [reply README.md:112]: sorry !!! add the header ```#import "<yourProjectName>-Swift.h"```. And sometimes close the xcode and reopen it if doesn't compile

**Comment by jrwpatterson** [comment]
is this going in I was working on a PR myself.

**Comment by dmitry-blackwave** [comment]
Hello! Sorry for the late answer. I'll check it and merge

**Pull request merged**